src/lib.rs

```diff
@@ -201,5 +225,7 @@ impl LoggerUi {
 /// Draws the logger ui
 /// has to be called after [`init()`](init());
 pub fn logger_ui(ui: &mut egui::Ui) {
-    (*LOGGER_UI.lock().unwrap()).ui(ui);
+    if let Ok(ref mut logger_ui) = LOGGER_UI.lock() {
+        logger_ui.ui(ui);
+    }
```

Comment:
So when it fails to lock, the ui won't be shown? 
I would guess that this shouldn't happen but might be nice to add a label with "Something went wrong loading the log" or similar

Reply:
```suggestion
    if let Ok(ref mut logger_ui) = LOGGER_UI.lock() {
        logger_ui.ui(ui);
    } else { 
        ui.colored_label(Color32::RED, "Something went wrong loading the log");
    }
```

Reply:
Yeah this looks good, It's worth noting that the `Err` should only appear if another thread panicked while holding the lock on the mutex. Since this is a private variable only exposed through the function which renders the UI, the only other thread which might cause an error is from egui. I.e. if this mutex fails to lock it means a panic happened in the UI, which probably means the app will crash.

I'm unsure how egui uses threads so this seems like a good idea, no real downside 👍